app/models/organization.rb

```diff
@@ -2,7 +2,6 @@
 
 class Organization < ApplicationRecord
   validates :name, presence: true, uniqueness: true
-  has_many :people, dependent: :destroy
```

Comment:
Wow, dat dat er nog in zat. Vet brak eigenlijk. People hebben al lang geen organization_id meer.